scripts/push-vellum-workflows.ts

```diff
@@ -168,6 +179,114 @@ class VellumWorkflowPusher {
+      
+      const newTag = this.incrementPatchVersion(currentTag);
+      console.log(`📦 Pushing new container image with tag: ${newTag}`);
+      
+      const dockerfilePath = join(this.agentDir, "src", "workflows", "Dockerfile");
+      const pushImageCommand = `poetry run vellum images push vargasjr:${newTag} --source ${dockerfilePath}`;
+      
+      execSync(pushImageCommand, {
+        cwd: this.agentDir,
+        stdio: 'inherit',
+        env: {
+          ...process.env,
+          VELLUM_API_KEY: process.env.VELLUM_API_KEY
+        }
+      });
+      
+      console.log(`✅ Successfully pushed container image: vargasjr:${newTag}`);
+      
+      this.updateLockFileTag(lockFileContent, newTag);
+      writeFileSync(lockFilePath, JSON.stringify(lockFileContent, null, 2));
+      console.log(`📝 Updated vellum.lock.json with new tag: ${newTag}`);
+      
+      console.log(`🔄 Retrying workflow push for: ${workflowName}`);
+      const retryResult = await this.pushWorkflow(workflowName);
+      
+      if (retryResult.success) {
+        console.log(`✅ Successfully pushed workflow after image update: ${workflowName}`);
+      }
+      
+      return retryResult;
+      
+    } catch (error: any) {
+      const errorMessage = `Failed to handle SDK version mismatch for ${workflowName}: ${error}`;
+      console.error(`❌ ${errorMessage}`);
+      return { success: false, error: errorMessage };
+    }
+  }
+
+  private incrementPatchVersion(version: string): string {
+    const parts = version.split('.');
+    if (parts.length !== 3) {
+      throw new Error(`Invalid version format: ${version}`);
+    }
+    
+    const major = parseInt(parts[0]);
+    const minor = parseInt(parts[1]);
+    const patch = parseInt(parts[2]) + 1;
+    
+    return `${major}.${minor}.${patch}`;
+  }
+
+  private updateLockFileTag(lockFileContent: any, newTag: string): void {
+    if (lockFileContent.workflows && Array.isArray(lockFileContent.workflows)) {
+      lockFileContent.workflows.forEach((workflow: any) => {
+        if (workflow.container_image_name === "vargasjr") {
+          workflow.container_image_tag = newTag;
+        }
+      });
+    }
+  }
+
+  private async handleLockFileChanges(): Promise<void> {
+    try {
+      const gitStatus = execSync('git status --porcelain agent/vellum.lock.json', { 
+        encoding: 'utf8',
+        cwd: process.cwd()
+      }).trim();
+      
+      if (gitStatus) {
+        console.log("🔍 Detected changes in vellum.lock.json, creating PR...");
+        
+        const timestamp = Math.floor(Date.now() / 1000);
+        const branchName = `devin/${timestamp}-update-vellum-lock-file`;
+        
+        execSync(`git checkout -b ${branchName}`, { cwd: process.cwd() });
+        execSync('git add agent/vellum.lock.json', { cwd: process.cwd() });
+        execSync('git commit -m "Update vellum.lock.json with new container image tag"', { cwd: process.cwd() });
```

Comment:
Update commit message to say "with new workflow changes" instead of "with new container image tag"

Reply:
Do this same copy change for any of the PR copy below